frontend/src/components/BuySellAsset.vue

```diff
@@ -172,19 +192,15 @@ const isValid = computed<boolean>(() => {
   return true;
 });
 
-const save = async (type: 'buy' | 'sell' | undefined) => {
+const save = async () => {
   if (!isValid.value) {
     return;
   }
 
-  if (transactionInternal.value.isNew()) {
-    transactionInternal.value.type = type;
-  }
-
   transactionInternal.value.totalExcludingFees =
     transactionInternal.value.price * transactionInternal.value.quantity;
 
-  transactionInternal.value.currency = transactionInternal.value.symbol.currency ?? 'USD';
+  transactionInternal.value.currency = transactionInternal.value.symbol?.currency ?? 'USD';
```

Comment:
We might want to use the Account default currency here instead of USD

Reply:
Do we alteady save that information?

Reply:
Yep https://github.com/GoPlan-Finance/GoPlan-app/blob/master/common/src/models/Account.ts#L19

Reply:
And i knew it 😉